klyshko-provisioner/kii-provisioner-run.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2022 - for information on the respective copyright owner
+# see the NOTICE file and/or the repository https://github.com/carbynestack/klyshko.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+# Fail, if any command fails
+set -e
+
+mkdir -p ~/.cs
+cat <<EOF > cs-config
+{
+  "prime" : 0,
+  "r" : 0,
+  "noSslValidation" : true,
+  "trustedCertificates" : [ ],
+  "providers" : [ {
+    "amphoraServiceUrl" : "http://ignore",
+    "castorServiceUrl" : "http://cs-castor:10100",
+    "ephemeralServiceUrl" : "http://ignore",
+    "id" : 1,
+    "baseUrl" : "http://ignore"
+  }, {
+    "amphoraServiceUrl" : "http://ignore",
+    "castorServiceUrl" : "http://ignore",
+    "ephemeralServiceUrl" : "http://ignore",
+    "id" : 2,
+    "baseUrl" : "http://ignore"
+  } ],
+  "rinv" : 0
```

Comment:
Not really needed for the json, but maybe move this up, so that all MP-SPDZ variables are together

Reply:
Makes sense. Done.